src/components/ReviewCard/index.js

```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import styled from "styled-components";
+import useSWR from "swr";
+import useSWRMutation from "swr/mutation";
+import DeleteButton from "../DeleteButton";
+import EditButton from "../EditButton";
+import Spinner from "../Spinner";
+
+const StyledReviewCard = styled.li`
+  background: #f0f0f0;
```

Comment:
Maybe this color could be added to your :root selector in the styles file

Reply:
True! 💪There is a big styling/refactoring update coming soon. I just recently learned a few tricks from your last PR that I'm going to implement 🤩👌

Reply:
I am really pleased to read that ☺️